includes/media_bynder.admin.inc

```diff
@@ -65,7 +65,7 @@ function media_bynder_save_image($id, $idHash){
         return array('success' => 0, 'message' => $rest_client->error);
     }
 
-    $uri = file_stream_wrapper_uri_normalize("bynder://f/$id/i/$idHash");
+    $uri = file_stream_wrapper_uri_normalize($arr_json['thumbnails']['thul']);
```

Comment:
Are you sure about this? This means you won't save the image ID to drupal, but the source..